whipper/common/mbngs.py

```diff
@@ -182,7 +182,10 @@ def _getMetadata(release, discid, country=None):
 
     discMD = DiscMetadata()
 
-    discMD.releaseType = release['release-group']['type']
+    if 'type' in release['release-group']:
+        discMD.releaseType = release['release-group']['type']
+    else:
+        discMD.releaseType = u''
```

Comment:
These last two lines could be removed and `discMD.releaseType` would default to be `None`. However, `command.mblookup` has `'    Type:   %s' % md.releaseType.encode('utf-8')` which will break if `releaseType` is `None` due to a missing function. This could potentially easily be fixed by either moving the `.encode()` method call or by typecasting `md.releaseType`. I may make a follow up commit later that does this, I just haven’t decided yet whether this would be something that needs a test (if there isn’t one already), since it is basically "just" a refactor. :)